lib/conjur/debify.rb

```diff
@@ -452,12 +476,23 @@ def build_test_image(appliance_image_id, project_name, package_name)
   c.flag [ :t, "image-tag"]
 
   c.desc "Bind another source directory into the container. Use <src>:<dest>, where both are full paths."
-  c.flag [ :"bind" ]
+  c.flag [ :"bind" ], :multiple => true
   
   c.desc "'docker pull' the Conjur container image"
   c.default_value false
   c.switch [ :pull ]
 
+  c.desc "Specify link for container"
+  c.flag [ :l, :link ], :multiple => true
+
+  c.desc 'Run dev-install in /src/<project-name>'
+  c.default_value true
+  c.switch [:'dev-install']
```

Comment:
This seems like a reasonable convenience. @kgilpin: was there a reason why you didn't do this?